src/cli_utils.rs

```diff
@@ -151,3 +161,57 @@ where
         IDENT_INACCURATE
     )
 }
+
+pub struct FormattableStats<'a, F: NumFormat> {
+    stats: &'a Stats,
+    num_format: &'a F,
+}
+
+impl<'a, F: NumFormat> FormattableStats<'a, F> {
+    pub fn new(stats: &'a Stats, num_format: &'a F) -> Self {
+        FormattableStats { stats, num_format }
+    }
+}
+
+macro_rules! display_stats {
```

Comment:
I know this is code that I wrote and not yours, but this really shouldn't be a macro since you should be able to easily write this as a function instead.

Reply:
I agree 😄 I was wondering why it was in a macro to begin with. I'll move it out.